### PR TITLE
Update rusk

### PR DESCRIPTION
**Pull request by CPerezz**
Resolves: #292 

Idk if I should bump rusk. Due to the dep change and all that stuff.. So whatever you think is the best I'll do it. I don't think it makes sense yet TBH.

### CODE REVIEW

File: rusk/src/lib/services/blindbid.rs [hunk 1]
```diff
@@ -27,6 +27,8 @@ pub use rusk_proto::blind_bid_service_server::{
     BlindBidService, BlindBidServiceServer,
 };
 
+pub(crate) const BLINDBID_TRANSCRIPT_INIT: &'static [u8] = b"BlindBidProof";
```
Comment (vlopes11) [inline rusk/src/lib/services/blindbid.rs:30]: Shouldn't be `b"dusk-network"`?

File: rusk/src/lib/services/blindbid/score_gen_handler.rs [hunk 2]
```diff
@@ -110,11 +110,12 @@ fn parse_score_gen_params(
...
-        .expect("Failed to get blindbid circuit keys from rusk_profile.");
+    let pk =
+        rusk_profile::keys_for(&BlindBidCircuit::CIRCUIT_ID)?.get_prover()?;
 
-    let prover_key = ProverKey::from_bytes(&prover_key[..])?;
+    let prover_key = ProverKey::from_slice(&pk)?;
     // Generate a proof using the circuit
-    circuit.gen_proof(&crate::PUB_PARAMS, &prover_key, b"BlindBid")
+    circuit
+        .gen_proof(&crate::PUB_PARAMS, &prover_key, b"BlindBidProof")
```
Comment (vlopes11) [inline rusk/src/lib/services/blindbid/score_gen_handler.rs:119]: Use `BLINDBID_TRANSCRIPT_INIT`

File: rusk/src/lib/services/blindbid/score_gen_handler.rs [hunk 3]
```diff
@@ -110,11 +110,12 @@ fn parse_score_gen_params(
...
+        .map_err(|e| anyhow::anyhow!("{:?}", e))
```
Comment (vlopes11) [inline rusk/src/lib/services/blindbid/score_gen_handler.rs:120]: I believe anyhow is being systematically removed from the stack

**Review by vlopes11 — APPROVED** [review]
LGTM, small optional nits

File: rusk/src/lib/services/blindbid.rs (hunk 1 above)
Comment (CPerezz) [reply rusk/src/lib/services/blindbid.rs:30]: Right!!!!

File: rusk/src/lib/services/blindbid/score_gen_handler.rs (hunk 3 above)
Comment (CPerezz) [reply rusk/src/lib/services/blindbid/score_gen_handler.rs:120]: Will do it in a follow up PR. Now BlindBid service is not even used nor tested. So I'm not so concerned about it.

**Review by ZER0 — APPROVED** [review]
LGTM with the comments from @vlopes11 addressed.

Also, we might want to add a CHANGELOG sooner rather than later.

**Pull request merged**